rolebinding-dev-namespace.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: dev-user-access
+  namespace: dev
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: dev-user-full-access
+subjects:
+- kind: Group
+  namespace: dev
+  name: 6301165d-4aac-45d5-a8b5-373bc6520e2f
```

Comment:
These create the rolebindings that can be used to limit access via RBAC. Groups must be created with azure cli or some other way.